Contest_API.md

```diff
@@ -1126,6 +1126,9 @@ Properties of problem objects:
 | rgb               | string ?  | Hexadecimal RGB value of problem color as specified in [HTML hexadecimal colors](https://en.wikipedia.org/wiki/Web_colors#Hex_triplet), e.g. `#AC00FF` or `#fff`.
 | color             | string ?  | Human readable color description associated to the RGB value.
 | time\_limit       | number    | Time limit in seconds per test data set (i.e. per single run). Should be a non-negative integer multiple of `0.001`. The reason for this is to not have rounding ambiguities while still using the natural unit of seconds.
+| memory\_limit     | integer   | Memory limit in MiB enforced on a submission.
+| output\_limit     | integer   | Limit in MiB on what the submission can write both to `stdout` and `stderr`. If a submission produces more output, a CCS should fail the submission or ignore output beyond this limit.
```

Comment:
We should pick one: fail submission or ignore output, or leave them off altogether like we do  for `time_limit` and `memory_limit `(that is, we don't specify what happens for time_limit exceeded, but we "know" what happens).  I think if output limit is exceeded, it should fail, even for `stderr`.

Reply:
More importantly (?) This behavior should not be **specified** in the Contest API (but it can be described). I believe it is specified in the problem format, and maybe that's enough, but otherwise it should probably go in the CCS reqs?